app/models.py

```diff
@@ -1906,6 +1910,7 @@ def serialize(self) -> dict:
             "postcode": None,
             "type": self.notification_type,
             "status": self.get_letter_status() if self.notification_type == LETTER_TYPE else self.status,
+            "status_description": self.formatted_status,
```

Comment:
Did we want to wrap this in the feature flag as well, or maybe we dont care?

Reply:
I think it's okay without a feature flag - I believe the plan is just to release it right away.